src/test/java/hw2/ex1/Ex1.java

```diff
@@ -0,0 +1,94 @@
+package hw2.ex1;
+
+import hw2.MainTestClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Ex1 extends MainTestClass {
```

Comment:
1. Why do you make this task as several test?
2. Are you aware that your assertion does not work in these tests and you will have always **PASSED** status for them?

Reply:
1. This tests verify different parts of web page, that's why i used several tests. I think it's correct.  
2. There was request to use soft asserts in task, so i did it.

Reply:
> This tests verify different parts of web page, that's why i used several tests. I think it's correct.

We discussed on the last class that this approach does not correct

> There was request to use soft asserts in task, so i did it.

What is required statement for the soft assert?